pkglistgen/tool.py

```diff
@@ -717,6 +719,22 @@ def update_and_solve_target(
         if not product_version:
             # for stagings the product version doesn't matter (I hope)
             product_version = '1'
+        with open(product_dir + "/release.changes", 'w', encoding="utf-8") as f:
+            try:
+                date = datetime.strptime(product_version, "%Y%m%d")
+            except ValueError:
+                logging.warning(
+                    "Warning: This should only happen in staging. Can not parse product version %s as date, "
```

Comment:
This is also used for ALP, BCI, Leap, SLE, ... which do not use a date as version.

Reply:
That is a problem...